cmd/update-index/main.go

```diff
@@ -165,7 +165,10 @@ func main() {
 	fs := flag.NewFlagSet("arguments", flag.ExitOnError)
 	fs.StringVar(&args.templateFile, "index-template", "./cmd/update-index/data/index.html.template", "path to the index.html template file")
 	fs.StringVar(&args.outputFile, "index-output", "./dist/index.html", "the location of the file this program writes")
-	fs.Parse(os.Args[1:])
+	err := fs.Parse(os.Args[1:])
+	if err != nil {
+		log.Fatalf("Failed to parsing the flags: %v", err)
```

Comment:
Maybe `errors.Wrap()` instead?

Reply:
this is part of the main function, right now it will log the error and then return with `os.exit(1)` 
I think we can change this in a follow-up and avoid `log.fatal` and panics.

This PR does not have the intention to refactor, those changes were made only to satisfy the lints 😄 
I will be glad to open follow up PRs to improve the code a bit